.github/workflows/ci.yml

```diff
@@ -17,4 +17,4 @@ jobs:
         with:
           go-version: '1.13.12'
```

Comment:
Why older version of go?

Reply:
Because I did not change it. I prefer to use 1.16.5.

Reply:
I think we should use 1.16 but we need to make sure this will be compatible with go-spacemesh build min version.